frontend/src/styles/HetComponents/HetPaginationButton.tsx

```diff
@@ -15,21 +15,24 @@ export default function HetPaginationButton(props: HetPaginationButtonsProps) {
   return (
     <Button
       onClick={props.onClick}
-      className='m-5 mt-auto flex md:w-full sm:w-auto flex-col justify-center rounded-3xl bg-methodologyGreen font-sansTitle  font-medium leading-lhSomeMoreSpace tracking-wide text-altBlack shadow-raised-tighter hover:shadow-raised lg:w-80 max-h-32 h-32'
+      className='m-5 mt-auto lg:mb-0 flex md:w-full sm:w-auto flex-col justify-center rounded-3xl bg-methodologyGreen font-sansTitle  font-medium leading-lhSomeMoreSpace tracking-wide text-altBlack shadow-raised-tighter hover:shadow-raised lg:w-80 min-h-24 max-h-32 h-24'
     >
       {/* ARROW AND DIRECTION WORD */}
       <span
-        className={`mt-5 flex items-center self-stretch font-sansText text-text leading-lhLoose text-altBlack lg:leading-lhSomeMoreSpace ${
+        className={`mt-5 flex items-center self-stretch font-sansText text-small leading-lhLoose text-altBlack lg:leading-lhSomeMoreSpace ${
           isPrevious ? 'ml-5 justify-start' : 'mr-5 justify-end'
         }`}
       >
         {isPrevious ? (
-          <span>
-            <ArrowBack /> <span>Previous</span>
+          <span className='flex align-center'>
+            <ArrowBack /> <span >Previous</span>
           </span>
         ) : (
-          <span>
+          <span className='flex align-center'>
             <span>Up Next</span> <ArrowForward />
+            <span className='text-sansText'>
+arrow_right_alt
+</span>
```

Comment:
I think this might have been left in my mistake.

Reply:
removed! thanks for catching that